server/channels/app/plugin_signature.go

```diff
@@ -90,8 +90,10 @@ func (ch *Channels) verifyPlugin(plugin, signature io.ReadSeeker) *model.AppErro
 			continue
 		}
 		publicKey := bytes.NewReader(pkBytes)
-		plugin.Seek(0, 0)
-		signature.Seek(0, 0)
+		//nolint:errcheck // safe operation: setting offset to the start of the reader
+		plugin.Seek(0, io.SeekStart)
+		//nolint:errcheck // safe operation: setting offset to the start of the reader
+		signature.Seek(0, io.SeekStart)
```

Comment:
Why are you not checking the error that `Seek` returns?

Reply:
It seemed to be a safe operation. I assumed that the "0" offset does not exceed the seize of the underlying object.
But I decided to log errors and continue the for-loop.